cmd/mailcatch/http.go

```diff
@@ -0,0 +1,124 @@
+package main
```

Comment:
These files should live under `test/` with the other test servers and should follow their naming scheme (`dns-test-srv`/`ct-test-srv`/`mail-test-srv`)